packages/use-contractkit/src/constants.tsx

```diff
@@ -84,7 +84,13 @@ export const PROVIDERS: {
   },
   [SupportedProviders.MetaMask]: {
     name: SupportedProviders.MetaMask,
-    description: (
+    description: isMobile ? (
+      isEthereumFromMetamask() ? (
```

Comment:
Newbie question, why are there 2 scenarios with Metamask? Trying to understand what `isEhtereumFromMetamask` checks.

Reply:
If you are on a mobile phone in a web browser vs if you are on a mobile phone in the metamask mobile app (`window.etherem.metmask` will be available).